src/sql/workbench/services/query/common/query.ts

```diff
@@ -9,6 +9,7 @@ export interface IColumn {
 	columnName: string;
 	isXml?: boolean;
 	isJson?: boolean;
+	columnSize?: number;
```

Comment:
I couldn't see where this is being set?  is it an existing field on the concrete class that your now exposing in the interface or something else?

Reply:
Yes, that's an existing field on the concrete class that I'm now exposing in the interface. I needed it to calculate the column widths.